Formula/kubernetes-service-catalog-client.rb

```diff
@@ -0,0 +1,33 @@
+class KubernetesServiceCatalogClient < Formula
+  desc "Consume Services in k8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
```

Comment:
Looks like we're up to `0.1.28` now, if you wouldn't mind catching that.

Reply:
Updated to 0.1.28. Thanks for being thorough.